src/game/objects/pickup.c

```diff
@@ -178,6 +197,7 @@ void Pickup_CollisionControlled(
             && lara_item->current_anim_state == AS_PICKUP) {
             if (lara_item->frame_number == AF_PICKUP_ERASE) {
                 PickUp_GetItem(item_num, item, lara_item);
+                PickUp_GetMultiple(item_num, item, lara_item);
```

Comment:
Can we refactor it a bit?
Instead of calling two functions, call 1, `PickUp_GetAllAtLaraPos(lara_item);` and then 
```c
static void PickUp_GetAllAtLaraPos(ITEM_INFO *lara_item)
{
    int16_t pickup_num = g_RoomInfo[lara_item->room_number].item_number;
    while (pickup_num != NO_ITEM) {
        ITEM_INFO *check_item = &g_Items[pickup_num];
        if (item->pos == lara_item->pos && g_Objects[check_item->object_number].collision == Pickup_Collision) {
            PickUp_GetItem(pickup_num, check_item, lara_item);
        }
        pickup_num = check_item->next_item;
    }
}
```

Reply:
Tested a slight variation for multi and single pickups above and underwater with walk to items on / off. Works well.